### ISSUE TITLE
a11y: text and question tasks no longer receive focus after Next or Back

### ISSUE THREAD

**Issue by eatyourgreens** [zooniverse/front-end-monorepo#7275]
## Package

- lib-classifier

## Describe the bug
Since #7218 merged, text boxes and question tasks at the beginning of a workflow step no longer receive focus after pressing Next or Back.

## To Reproduce
- https://www.zooniverse.org/projects/bluejackets/civil-war-bluejackets/classify/workflow/25873/subject-set/134666
- Tab to the question and select No.
- Tab to Next and press it.
- you now have to press Tab three times before you can enter any text.
- Tab to Back and press it.
- You now hav to Tab three times (through the tab list) to reach the selected answer radio button.

You can restore previous behaviour, for comparison, with:
```sh
git checkout c9022cdb4^
```

## Expected behavior
When the first task in a step is a text box, that text box should be focused after pressing either Next or Back. Ideally, the first task in any step should be focused after pressing Next or Back, but the new focus behaviour could be used as a fallback, when there's no focus management specified for a task component.

Here's a screen recording of the previous behaviour, from Civil War Bluejackets. I'm navigating the workflow using only my keyboard. Notice how the text task is focused after I press Next, and the question task is focused after I press Back. This has been the behaviour of the task area for about five years now (longer if you include PFE behaviour, which this mirrors.)

https://github.com/user-attachments/assets/363ce3b7-b549-462d-91c4-33a998782161

**NB.** this video also shows another WCAG failure. When the first question has focus, there's no visible focus indicator (#6871.)

In this specific example, the form labels for both tasks are overly verbose and would likely fail [WCAG 3.3.2 labels & instructions](https://www.w3.org/WAI/WCAG22/Understanding/labels-or-instructions.html), which requires form labels to be concise and clear.

## Additional context
Some Talk discussions where transcription volunteers, who rely on the keyboard, are asking for the old behaviour to be restored:
- https://www.zooniverse.org/talk/17/3935557
- https://www.zooniverse.org/talk/14/3948584

There might be other posts on individual transcription projects too.

Historical context: I added the autofocus behaviour for HMS NHS and Scarlets & Blues, since both Royal Museums Greenwich & National Archives [are legally required to meet WCAG AA](https://www.gov.uk/guidance/accessibility-requirements-for-public-sector-websites-and-apps#meeting-accessibility-requirements) for their transcription projects. I don't think the new focus behaviour meets that standard.

**Comment by eatyourgreens** [comment]
TBH skip links would be the accessible solution to quickly reach the classifier when the Classify page first loads. 

If the page is properly marked up with landmark elements, then screen readers can also use landmark navigation to quickly reach the classifier. Labelled `<section>` elements could help here, as they create named region landmarks within the main content.
